benchmark/uintvar.cpp

```diff
@@ -20,9 +20,9 @@ UIntVar_FromUint64(benchmark::State& state)
 static void
 UIntVar_ToUint64(benchmark::State& state)
 {
-    constexpr auto var_int = quicr::UintVar(0x123456789);
+    constexpr auto kVarInt = quicr::UintVar(0x123456789);
```

Comment:
Wow, is it nit-picky. Even for a local constexpr?

Reply:
Theoretically I think this *is* allowed either way in the guide:

> All such variables with static storage duration (i.e., statics and globals, see Storage Duration for details) should be named this way, including those in templates where different instantiations of the template may have different values. This convention is optional for variables of other storage classes, e.g., automatic variables; otherwise the usual variable naming rules apply. For example:

But our rules aren't quite setup for that. Let me see if I can get there.

Reply:
But these aren't static? It's fine, just thought it was weird it would care about a local looking like that.

Reply:
Sorry, that's what I'm saying. These don't need the k prefix because they're not static. But our clang tidy rules that we gave setup do not match the guide quite at that level of granularity, so they're sort of giving a false positive.

Reply:
Ah, makes sense. Thanks!